### DISCUSSION TITLE
WireGuard: configuration not enabled after LXC-reboot

### DISCUSSION THREAD

**Discussion by Kroesss** [tteck/Proxmox#3790] [Help Needed]
So I'm having a little bit of weird behaviour on my WireGuard LXC-container. In general, it works great. I only noticed that the connection wasn't working anymore after I had updated Proxmox, and rebooted after that (there was a kernel-update, and a pop-up suggested rebooting the complete node, so I did).

After a little bit of looking around to check if the WireGuard LXC still had the correct IP assigned and so on, I saw that the configuration was turned off. A simple click on the on/off switch in WGDashboard fixed it. I then tried rebooting the LXC itself, and it came back up with the configuration disabled.

I would expect the configuration to be set to the same state as it was when the reboot was initiated. This way, I will never be able to restart the container or the node when away, and when I do reboot, I have to remember to actively switch WireGuard back on afterwards.

Is there a solution for this? I only have 1 configuration, and I'd like that to be on by default after a reboot.

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/3632#discussioncomment-10592708
